js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,24 @@
+const deepCopyObject = value => {
+  if (isPrimitive(value)) {
+    return value;
+  }
+
+  return deepCopyOfObject(value);
+};
+
+function deepCopyOfObject(object) {
+  let deepCopyOfObject = {};
+
+  for (const property in object) {
```

Comment:
As explained in today's session, avoid for-in.

Reply:
Sure